.github/workflows/ci.yml

```diff
@@ -122,6 +122,9 @@ jobs:
               run: 'pnpm --filter="${{ matrix.projectName }}" ${{ matrix.testEnv.start }}'
 
             - name: 'Run tests'
+              env:
+                  BUILDKITE_ANALYTICS_TOKEN: ${{ secrets.BUILDKITE_ANALYTICS_TOKEN }}
+                  BUILDKITE_ANALYTICS_MESSAGE: ${{ github.event.pull_request.title }}
```

Comment:
For push event this will fail.

Reply:
Lika a push to the branch?  Or do you mean before a PR is created from a branch?  In testing this out, any subsequent pushes I made just used the title of the original PR.

Reply:
I get it.  Because `ci.yml` is used in workflows other than just the PR one, it could potentially get called and there is no pull request title.

Buildkite recommends commit message... so I'll use that instead.